pkg/validations/cluster.go

```diff
@@ -215,3 +217,12 @@ func ValidateK8s129Support(clusterSpec *cluster.Spec) error {
 	}
 	return nil
 }
+
+// ValidateEksaReleaseExistOnManagement checks if there is a corresponding eksareleases CR for workload's eksaVersion on the mgmt cluster.
+func ValidateEksaReleaseExistOnManagement(ctx context.Context, k KubectlClient, mgmtCluster *types.Cluster, workload *cluster.Spec) error {
+	v := workload.Cluster.Spec.EksaVersion
+	if release, err := k.GetEKSARelease(ctx, string(*v), mgmtCluster.KubeconfigFile); err != nil || release == nil {
```

Comment:
these seem like they should be two different errors imo (error reading and not found error)